CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## VNext
 
+
+## Version 2.21.0
```

Comment:
do we not want to wait for some confirmation that https://github.com/microsoft/ApplicationInsights-dotnet/issues/1186#issuecomment-1161971528 is fixed?

Reply:
It's been almost a month and no one has commented on that thread. 
We have a security fix in this release that codeless needs to take.
I'll leave that issue open until we get confirmation, but want to go ahead and prep the release for Wed.